server/models/event.js

```diff
@@ -15,20 +15,37 @@ const eventSchema = new mongoose.Schema({
         type: String,
         required: true
     },
+    city: {
+        type: String,
+        required: true
+    },
+    state: {
+        type: String,
+        required: true
+    },
     website: {
         type: String
     },
     details: {
         type: String,
         required: true
     },
-    organization: {
-        type: String
-    }
+    user: [
+        {
+           type: mongoose.Schema.Types.ObjectId,
+           ref: 'User' 
+        }
+    ],
+    organization: [
+        {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'Organization'
+        }
+    ]
```

Comment:
When it comes to Events, this is a small thing, but they should only have a 1:M relationship with organization, I think, because only one organization is going to be hosting an Event, not multiple of them.